### ISSUE TITLE
Please consider remove /etc/grub2-efi.cfg symbolic link

### ISSUE THREAD

**Issue by vincentmli** [openeuler-mirror/community#1]
## Proposal

**General Information**

openEuler-21.09-x86_64.qcow2.xz KVM image has `/etc/grub2-efi.cfg` that is symbolic link to `/boot/grub2/efi/EFI/openEuler/grub.cfg`, users might need to use grubby to change kernel boot parameters like below

```
grubby --grub2 --args="apparmor=0" --remove-args="rhgb quiet" --update-kernel /boot/vmlinuz-5.10.0-5.10.0.24.oe1.x86_64
```
to run light weight k3s kubernetes cluster due to k3s issue https://github.com/rancher/k3os/issues/702,  but above grubby command only changed `/etc/grub2-efi.cfg`, ignored `/etc/grub2.cfg` which is symbolic link to `/boot/grub2/grub.cfg` that the VM will actually use unless users specifically configured VM to use EFI to boot the VM https://fedoraproject.org/wiki/Using_UEFI_with_QEMU, which I think most of users don't do that.

https://docs.fedoraproject.org/en-US/fedora/rawhide/system-administrators-guide/kernel-module-driver-configuration/Working_with_the_GRUB_2_Boot_Loader/#sec-Making_Persistent_Changes_to_a_GRUB_2_Menu_Using_the_grubby_Tool

 I recommend to either remove `/etc/grub2-efi.cfg` or maybe documents it somewhere about how to change the VM kernel boot parameter according to if EFI configured or not in VM qemu.

 
**How to reproduce the issue**

1. Download  https://repo.openeuler.org/openEuler-21.09/virtual_machine_img/x86_64/openEuler-21.09-x86_64.qcow2.xz and ran the openEuler image in KVM,

2. `grubby --grub2 --args="apparmor=0" --remove-args="rhgb quiet" --update-kernel /boot/vmlinuz-5.10.0-5.10.0.24.oe1.x86_64` 
3. `reboot`
4. `cat /proc/cmdline` to check if kernel has `"apparmor=0"` set

**Comment by Yikun** [comment]
hi, this is only a mirror repo, pls see https://gitee.com/openeuler/community

**Comment by vincentmli** [comment]
@Yikun 

Yes i am aware of https://gitee.com/openeuler/community, but it appears only for Chinese audience who can read Chinese, what about reaching to users who can't read Chinese? I hope openEuler can reach to  users can't read Chinese.

**Comment by Yikun** [comment]
@vincentmli You can switch language to `English` at right and end of page. I can also help to cherry pick this issue to gitee.

**Comment by Yikun** [comment]
https://gitee.com/openeuler/community/issues/I4L2HE
